EIPS/eip-2384.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: 2384
+title: Difficulty Bomb Delay
```

Comment:
May be it would make sense using a title following what the previous ones did (like 649): Istanbul Difficulty Bomb Delay or Berlin Difficulty Bomb Delay.

I understand it is unclear which one it will be part of though.

Reply:
ya, I can only update this one after it's decided.

Reply:
I'd argue that it can be called "Istanbul/Berlin Difficulty Bomb Delay" and clarified later. A generic "difficulty bomb delay" seems more confusing, given there are a bunch of other EIPs titled the same.